vignettes/spatial_wwinference.Rmd

```diff
@@ -0,0 +1,2457 @@
+    aes(
+      xmin = lower, xmax = upper,
+      y = inf_model_type,
+      colour = inf_model_type
+    ),
+    height = 0.2,
+    size = 1
+  ) +
+  geom_vline(
+    data = corr_matrix_df %>%
+      filter(model_type == "Exponential") %>%
+      mutate(
+        Row = Var1,
+        Column = Var2
+      ),
+    aes(xintercept = value),
+    color = "red",
+    linetype = "dashed",
+    size = 1
+  ) +
+  facet_grid(Row ~ Column, switch = "y") +
+  scale_fill_manual(
+    breaks = c("Exponential", "Unstructured"),
+    values = c("darkviolet", "deeppink3")
+  ) +
+  scale_colour_manual(
+    breaks = c("Exponential", "Unstructured"),
+    values = c("darkviolet", "deeppink3")
+  ) +
+  ylab("") +
+  xlab("Correlation Drawed") +
+  ggtitle("Data Generation Model : Exponential") +
+  labs(
+    color = "Assumed Corr. Structure",
+    fill = "Assumed Corr. Structure"
+  ) +
+  theme_bw() +
+  theme(
+    axis.text.y = element_blank(),
+    axis.ticks.y = element_blank()
+  )
+# ------------------------------------------------------------------------------
+# Rand. Corr. Matrix Correlation Matrix ----------------------------------------
+rand_corr_matrix_result_plot <- ggplot(all_corr_matrix_draws_df %>%
+  group_by(
+    Row,
+    Column,
+    inf_model_type,
+    gen_model_type
+  ) %>%
+  summarise(
+    lower = quantile(Freq, 0.025, na.rm = TRUE),
+    median = median(Freq),
+    upper = quantile(Freq, 0.975, na.rm = TRUE),
+    .groups = "drop"
+  ) %>%
+  filter(
+    gen_model_type == "Rand. Corr. Matrix",
+    inf_model_type != "IID"
+  )) +
+  geom_point(
+    aes(
+      x = median, y = inf_model_type,
+      colour = inf_model_type
+    ),
+    size = 2.5
+  ) +
+  geom_errorbarh(
+    aes(
+      xmin = lower, xmax = upper,
+      y = inf_model_type,
+      colour = inf_model_type
+    ),
+    height = 0.2,
+    size = 1
+  ) +
+  geom_vline(
+    data = corr_matrix_df %>%
+      filter(model_type == "Rand. Corr. Matrix") %>%
+      mutate(
+        Row = Var1,
+        Column = Var2
+      ),
+    aes(xintercept = value),
+    color = "red",
+    linetype = "dashed",
+    size = 1
+  ) +
+  facet_grid(Row ~ Column, switch = "y") +
+  scale_fill_manual(
+    breaks = c("Exponential", "Unstructured"),
+    values = c("darkviolet", "deeppink3")
+  ) +
+  scale_colour_manual(
+    breaks = c("Exponential", "Unstructured"),
+    values = c("darkviolet", "deeppink3")
+  ) +
+  ylab("") +
+  xlab("Correlation Drawed") +
+  ggtitle("Data Generation Model : Rand. Corr. Matrix") +
+  labs(
+    color = "Assumed Corr. Structure",
+    fill = "Assumed Corr. Structure"
+  ) +
+  theme_bw() +
+  theme(
+    axis.text.y = element_blank(),
+    axis.ticks.y = element_blank()
+  )
+# ------------------------------------------------------------------------------
+
+iid_corr_matrix_result_plot + exp_corr_matrix_result_plot +
+  rand_corr_matrix_result_plot +
+  plot_layout(
+    guides = "collect"
+  ) &
+  theme(legend.position = "bottom")
+```
```

Comment:
it looks to me like the prior on phi might still be a little too tight, as the exponential fit to the data generating iid model should in theory be able to estimate a (very high I think) phi which would revert back to an identity matrix. 

Interesting, and perhaps worrisome, that the exponential model appears to be estimating the randome correlation matrix better than the fully unstructured inference. I wonder if we just need to make the distances a bit more extreme to see more of a difference.